scipy/stats/tests/test_stats.py

```diff
@@ -2358,13 +2358,13 @@ def test_ttest_rel():
     assert_raises(ValueError, stats.ttest_rel, x, y, nan_policy='raise')
     assert_raises(ValueError, stats.ttest_rel, x, y, nan_policy='foobar')
 
+    # test zero division problem
+    t,p = stats.ttest_rel([0,0,0],[1,1,1])
```

Comment:
Also optional: spaces after commas please. There's no need to go haywire correcting PEP8 violations in random places, but once you've touched a line, it's nice to also improve readability.